si-id-epu50/si_id_epu50/epu_db.py

```diff
@@ -6,11 +6,13 @@
 #############################################
 # Map variable -> pv name
 ## general
-# pv_polarization_sel = 'Polarization-Sel'
-# pv_polarization_sts = 'Polarization-Sts'
-# pv_kparameter_sp = 'KParameter-SP'
-# pv_kparameter_rb = 'KParameter-RB'
-# pv_kparameter_mon = 'KParameter-Mon'
+pv_polarization_sel = 'Pol-Sel'
+pv_polarization_sts = 'Pol-Sts'
+pv_polarization_mon = 'Pol-Mon'
+pv_polarization_cmd = 'ChangePol-Cmd'
+pv_kparameter_sp = 'KParameter-SP'
+pv_kparameter_rb = 'KParameter-RB'
+pv_kparameter_mon = 'KParameter-Mon'
```

Comment:
Wouldn't we add equivalent PVs for Speed, MaxSpeed, AllowToChange, EnblAndRelease related to Polarization?